README.md

```diff
@@ -177,6 +180,7 @@ vous trouverez sur cette plateform des cours et examens qui vous accompagnerons
    * [codeforces](https://www.codechef.com/): best for competitive programming
    * [codingame](https://www.codingame.com/): Coding game and programming challenges to code better
    * [Timus Online Judge](http://acm.timus.ru/): An online archive for programming problems and automatic judging system.
+   * [Exercism](https://exercism.io/)
```

Comment:
We already listed Exercism under the` learn to code` category

```suggestion
```

Reply:
Please add a description to this link!